lerna.json

```diff
@@ -9,5 +9,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "0.81.0-beta.8"
+  "version": "0.82.0"
```

Comment:
As per the comments, this is a PR, not a release merge for release. Please revert the version bumps completely.

Reply:
To expand -

- we don't release until there has been sufficient testing of the beta package in apps (and there are some stuff that are still up-in-the air and may have impact of functionality, i.e. the contracts structure changes)

- we don't make api releases where the API depends on a non-stable common package tree (as it does here) - only stable versions of common for API stable releases

Reply:
Ok, I didn't know that. Happy to remove the version bumps, then. 
What exactly do you mean by:
>  non-stable common package tree

The fact that I still have the `polkadot-js/common` modules included as beta? So for a release, I would need to bump up all `@polkadot` modules to their next major version?

Reply:
The API is broken with the substrate master atm, though. So it could be good to get out a smaller release before getting all the stuff in that is still in the air.

Reply:
All the "stuff in the air" is merged already.